platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -199,14 +191,14 @@ public UsersWorkspaceDto getByName(@PathParam("name") String name) throws Server
     @Produces(APPLICATION_JSON)
     public RuntimeWorkspaceDto getRuntimeWorkspaceById(@PathParam("id") String id)
             throws ServerException, BadRequestException, NotFoundException {
-        return asDto(workspaceManager.getRuntimeWorkspace(id));
+        return asRuntimeWorkspaceDto(workspaceManager.getRuntimeWorkspace(id));
     }
 
     @GET
     @Path("/name/{name}/runtime")
     @Produces(APPLICATION_JSON)
     public RuntimeWorkspace getRuntimeWorkspaceByName(@PathParam("name") String name) {
-        return asDto(workspaceManager.getRuntimeWorkspace(name, getCurrentUserId()));
+        return asRuntimeWorkspaceDto(workspaceManager.getRuntimeWorkspace(name, getCurrentUserId()));
```

Comment:
The same [comment](https://github.com/codenvy/che-core/pull/307#discussion_r41075354)